algo/src/main/java/com/hartwig/actin/algo/evaluation/general/HasMaximumBMI.java

```diff
@@ -0,0 +1,69 @@
+package com.hartwig.actin.algo.evaluation.general;
+
+import com.hartwig.actin.PatientRecord;
+import com.hartwig.actin.algo.datamodel.Evaluation;
+import com.hartwig.actin.algo.datamodel.EvaluationResult;
+import com.hartwig.actin.algo.datamodel.ImmutableEvaluation;
+import com.hartwig.actin.algo.evaluation.EvaluationFactory;
+import com.hartwig.actin.algo.evaluation.EvaluationFunction;
+import com.hartwig.actin.clinical.datamodel.BodyWeight;
+import com.hartwig.actin.clinical.sort.BodyWeightDescendingDateComparator;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Optional;
+
+public class HasMaximumBMI implements EvaluationFunction {
+
+    static final String EXPECTED_UNIT = "kilogram";
```

Comment:
Why not use "Kilogram" here? Or is "kilogram" the typical value we get? 

Since we test ignoring case it does not really matter but still might be cleaner to put here what we generally expect?

Reply:
clear. 

Related, if you are not using `EXPECTED_UNIT` in test code you may as well make it private?